commit fe97f3329d55f2211b6f6d57a249f06714cdb9c8
Author: Johannes Berg <johannes.berg@intel.com>
Date:   Thu Apr 13 14:23:49 2017 +0200

    mac80211: fix MU-MIMO follow-MAC mode
    
    commit 9e478066eae41211c92a8f63cc69aafc391bd6ab upstream.
    
    There are two bugs in the follow-MAC code:
     * it treats the radiotap header as the 802.11 header
       (therefore it can't possibly work)
     * it doesn't verify that the skb data it accesses is actually
       present in the header, which is mitigated by the first point
    
    Fix this by moving all of this out into a separate function.
    This function copies the data it needs using skb_copy_bits()
    to make sure it can be accessed if it's paged, and offsets
    that by the possibly present vendor radiotap header.
    
    This also makes all those conditions more readable.
    
    Signed-off-by: Johannes Berg <johannes.berg@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1509_fe97f3329d55f2211b6f6d57a249f06714cdb9c8.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1509_fe97f3329d55f2211b6f6d57a249f06714cdb9c8.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
